### PR TITLE
merge new sign in and out feature

### PR DESCRIPTION
**Pull request by ronaldcohen1611**
I reimplemented the sign-in and out pages from the master.

### CODE REVIEW

**Comment by chrism1001** [comment]
reviewing right now

**Comment by chrism1001** [comment]
design looks great. will create a new branch to setup user contacts for logging in and registering.

**Comment by chrism1001** [comment]
image

File: web/src/Routes.tsx [hunk 1]
```diff
@@ -10,13 +10,21 @@
...
 
 import NewsLayout from 'src/layouts/NewsLayout'
+import SignInSignUpLayout from 'src/layouts/SignInSignUpLayout'
+
+import { useAuth } from './auth'
 
 const Routes = () => {
   return (
-    <Router>
+    <Router useAuth={useAuth}>
```
Comment (chrism1001) [inline web/src/Routes.tsx:19]: need a comment for what useAuth does. who is authorized to see these pages.

**Review by chrism1001 — APPROVED** [review]
just need unit testing.

File: web/src/Routes.tsx (hunk 1 above)
Comment (ronaldcohen1611) [reply web/src/Routes.tsx:19]: useAuth is the use hook that holds the state of whos currently logged in and manages logging out users from the database

**Comment by chrism1001** [comment]
image
Does anyone else get this issue with the end of line when opening up a file. I have to manually change it to LF every time. Is there a way we can make LF the default for everyone?

**Review by rodrigomv29 — APPROVED** [review]
Approved! I still have the problem with the LF. But other than that I think it looks good

**Comment by rodrigomv29** [comment]
I can work on the unit testing tonight.

**Pull request merged**